### DISCUSSION TITLE
Задачи с уведомлениями

### DISCUSSION THREAD

**Discussion by EKhorikov** [totumonline/totum-ru-issues-and-discussions#65] [Как это сделать?] [answered]
Как настроить задачи (например к заказам, заявкам, проектам, с уведомлениями в нотификациях, в случае их просрочки.
Пример из сервиса "МойСклад"
Создаём задачу в документе
1. ![image](https://user-images.githubusercontent.com/92679628/151700069-dfdaf356-418c-4630-8ec8-76e336ea5edd.png)
2. ![image](https://user-images.githubusercontent.com/92679628/151700406-1ff21481-ac09-4ac0-a76d-bf564362d09a.png)
3. ![image](https://user-images.githubusercontent.com/92679628/151700100-ea35ca8a-bb4f-4771-a398-ae059e4584e3.png)

После истечения указанного срока, появляется уведомление в нотификациях

4. ![image](https://user-images.githubusercontent.com/92679628/151700175-dfe62c4d-723f-4a05-8e84-9f85e00196eb.png)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
1. Создайте таблицу Задачи, наполните по своему желанию.
2. Проверяйте заданиями по расписанию просрочку.  про кроны: https://docs.totum.online/crons
3. Если выполняется условие просрочки — отправьте пользователю нотификацию:

https://docs.totum.online/notifications

https://docs.totum.online/notifications-course

вот здесь обсуждается как делать url-ссылки в нотификациях: https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/56#discussioncomment-2027089
